src/relentless/model/potential/bond.py

```diff
@@ -0,0 +1,425 @@
+import numpy
+
+from relentless import math
+
+from . import potential
+
+
+class BondParameters(potential.Parameters):
+    """Parameters for a bond potential."""
+
+    pass
+
+
+class BondPotential(potential.Potential):
+    r"""Abstract base class for a bond potential."""
+
+    pass
+
+
+class Harmonic(BondPotential):
+    r"""Harmonic bond potential.
+
+    .. math::
+
+        u(r) = \frac{k}{2} (r - r_0)^2
+
+    where :math:`r` is the distance between two bonded particles. The parameters
+    for each type are:
+
+    +-------------+--------------------------------------------------+-----------+
+    | Parameter   | Description                                      | Initial   |
+    +=============+==================================================+===========+
+    | ``k``       | potential constant                               |           |
+    +-------------+--------------------------------------------------+-----------+
+    | ``r0``      | rest length.                                     |           |
```

Comment:
Clean up these tables to look like the pair potentials:
```suggestion
    | ``k``       | Spring constant :math:`k`.                               |           |
    +-------------+--------------------------------------------------+-----------+
    | ``r0``      | Minimum-energy length :math:`r_0`.                                     |           |
```
Given that I don't think these potentials are going to have default parameters, I think you can also remove the `Initial` column.